test/commands/test-cli-errors.js

```diff
@@ -44,8 +44,11 @@ function errorIfBadCommand(command) {
 
 function errorIfNoStd(stds) {
   if (!stds || !Array.isArray(stds) || !stds.length > 0) {
-    throw new Error('testCli configuration error: "std" must be an not empty array ex.'
-      + ' [{in:\'john\'},{out:\'hello, john\'}]');
+    throw new Error('testCli configuration error: "std" or "exitCode" or "exitMessage" must be'
+      + ' defined.\n'
+      + ' "std" must be a not empty array like [{in:\'john\'},{out:\'hello, john\'}]\n'
+      + ' Define "exitCode" and/or "exitMessage" instead of "std" if you are testing an error'
+      + ' case.');
```

Comment:
[code style] I think that we tend to avoid string building/concatenation using `+`. You could replace it by backslashes.